### ISSUE TITLE
Monophonic sequences

### ISSUE THREAD

**Issue by riban-bw** [zynthian/zynthian-issue-tracking#877]
**Is your feature request related to a problem? Please describe.**
It would be fun to be able to create and edit monophonic sequences.

**Describe the solution you'd like**
An option to toggle between monophonic and polyphonic sequences or maybe a variable quantity of polyphony. Each time a note is added to a step in a pattern, any other notes on that step are removed. If we have variable polyphony then the oldest / lowest / highest notes beyond the current polyphony are removed.

**Describe alternatives you've considered**
Manually remove notes from pattern grid.

**Additional context**
A lot of step sequencers allow one note per step. This is particularly true in the analogue and module world. There is work to improve Zynthians integration with those worlds and some users may feel more comfortable with a monophonic sequencer. We also have monophonic instruments within Zynthian that may have undefined behaviour when fed with a polyphonic sequence.

**Comment by jofemodo** [comment]
It's implemented in the devel branch:

https://github.com/zynthian/zyncoder/tree/cvgateout_monophonic

**Comment by riban-bw** [comment]
No, this request is for monophonic sequences. That development was for monophonic CV/Gate.
